### PR TITLE
Open source pro

### PR DESCRIPTION
**Pull request by IgnisDa**
- [x] delete all daily user activities since there was a bug in summary calculation
- [x] change from cornflowerblue to `Paper withShadow`
- [x] add an `entity_ids` column in daily user activities that will store the IDs of the entities that contributed to that DAU
- [ ] ~allow editing workouts~ (#1046)

### CODE REVIEW

File: README.md
Comment (Jacob-Tate) [inline README.md:1]: I dont think you are intending to remove the logo from the readme

File: crates/services/importer/src/audiobookshelf.rs
Comment (Jacob-Tate) [inline crates/services/importer/src/audiobookshelf.rs:1]: Ill need to test this change to ensure the api url is correct

File: docs/content/architecture.md
Comment (Jacob-Tate) [inline docs/content/architecture.md:1]: Are you intending to remove this file?

File: docs/logo.png
Comment (Jacob-Tate) [inline docs/logo.png:1]: Are you intending to remove this file?

**Review by Jacob-Tate — COMMENTED** [review]
A few things I noticed

File: README.md
Comment (IgnisDa) [reply README.md:1]: I am. I plan to revisit it later.

File: crates/services/importer/src/audiobookshelf.rs
Comment (IgnisDa) [reply crates/services/importer/src/audiobookshelf.rs:1]: Sure. I tested it on my end too.

File: docs/content/architecture.md
Comment (IgnisDa) [reply docs/content/architecture.md:1]: I renamed it to contributing.md

File: docs/logo.png
Comment (IgnisDa) [reply docs/logo.png:1]: Yes. I found the orange color obnoxious lol

**Pull request merged**